wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder+MLS.swift

```diff
@@ -27,8 +27,8 @@ extension EventDecoder {
         Logging.mls.info("decrypting mls message")
 
         guard let decryptionService = context.mlsDecryptionService else {
-            WireLogger.mls.critical("failed to decrypt mls message: mlsDecyptionService is missing")
-            fatalError("failed to decrypt mls message: mlsService is missing")
```

Comment:
IMHO: we should replace all the fatalError to preconditionFailure. 

[details here](https://lapcatsoftware.com/articles/fatalError.html)